### PR TITLE
Create series for feeder podcast

### PR DESCRIPTION
**Pull request by kookster**
- [x] Retrieve feeder podcast
- [x] Create series record
- [x] copy images
- [x] create version template
- [x] create the distribution associated to feeder podcast

### CODE REVIEW

**Comment by gcampo88** [comment]
@kookster assume I should hold off on review for now? (Github's sending me mixed messages!)

File: lib/feeder_importer.rb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    self.distribution = Distributions::PodcastDistribution.create!(
+      url: feeder_podcast_url,
+      distributable: series,
+      audio_version_template: template
+    )
+
+    series
+  end
+
+  def retrieve_podcast(a_id = account_id, p_url = feeder_podcast_url)
```
Comment (gcampo88) [inline lib/feeder_importer.rb]: any particular reason for making the account id & podcast URL overrideable?

File: test/lib/feeder_importer_test.rb [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    stub_import_requests
+  end
+
+
+  it "makes a new importer" do
+    importer.wont_be_nil
+  end
+
+  it "retrieves the feeder podcast" do
+    podcast = importer.retrieve_podcast
```
Comment (gcampo88) [inline test/lib/feeder_importer_test.rb]: maybe name this version of `podcast` something slightly different to set it apart from the podcast you declared on line 13?

File: lib/feeder_importer.rb (hunk 1 above)
Comment (gcampo88) [reply lib/feeder_importer.rb]: & ditto for optional `podcast` parameter for `create_series` def

**Comment by kookster** [comment]
Realized I have tests to fix, assigned too early, sorry!

File: lib/feeder_importer.rb (hunk 1 above)
Comment (kookster) [reply lib/feeder_importer.rb]: yes, clearer to test if you can call the method without setting the state on the importer

Comment (gcampo88) [reply lib/feeder_importer.rb]: 👍

File: lib/feeder_importer.rb [hunk 3]
```diff
@@ -0,0 +1,95 @@
+require 'prx_access'
+require 'addressable/uri'
+
+class FeederImporter
+  include PRXAccess
+  include Announce::Publisher
+
+  attr_accessor :account_id, :user_id, :feeder_podcast_url
+  attr_accessor :podcast, :series, :template, :distribution
```
Comment (gcampo88) [inline lib/feeder_importer.rb]: assume these two `attr_accessor` calls are kept separate for organization's sake?

Comment (kookster) [reply lib/feeder_importer.rb]: yes, and line length would be > 100 if they were all on one line

Comment (gcampo88) [reply lib/feeder_importer.rb]: 👍

**Pull request merged**